ui/app/components/copy-button.js

```diff
@@ -1,10 +1,11 @@
 import Component from '@ember/component';
 import { task, timeout } from 'ember-concurrency';
-import { classNames } from '@ember-decorators/component';
+import { classNames, classNameBindings } from '@ember-decorators/component';
 import classic from 'ember-classic-decorator';
 
 @classic
 @classNames('copy-button')
+@classNameBindings('inset')
```

Comment:
`nit`: There are only 6 invocations of the `CopyButton` component. Rather than persisting `ember-classic` pattern, this seems like an easy win for us to migrate to Ember Octane via a Glimmer component!

Reply:
Roger dodger!